dataline-server/src/main/java/io/dataline/server/handlers/DestinationImplementationsHandler.java

```diff
@@ -68,15 +68,15 @@ public DestinationImplementationRead createDestinationImplementation(
     // validate configuration
     validateDestinationImplementation(
         destinationImplementationCreate.getDestinationSpecificationId(),
-        destinationImplementationCreate.getConnectionConfiguration());
+        (String) destinationImplementationCreate.getConnectionConfiguration());
```

Comment:
This is an assumption I am making and the code I read is unclear about it. Is the content of `getConnectionConfiguration` a string representing a json object or is it a JsonNode?

Reply:
it's a string. we should probably apply this same naming convention and type to the openapi config as well to clear up the confusion here. i think we should do that in a separate PR though so that we don't block this one.

Reply:
are we sure it will work with the frontend if we do that?